### PR TITLE
adv setting - payout (@eonwarped)

### PR DESCRIPTION
**Pull request by gl2748**
Inside org version of this PR:
https://github.com/steemit/condenser/pull/2879

### CODE REVIEW

File: src/app/components/modules/Modals.jsx [hunk 1]
```diff
@@ -39,6 +41,7 @@ class Modals extends React.Component {
         show_bandwidth_error_modal: PropTypes.bool,
         show_signup_modal: PropTypes.bool,
         show_promote_post_modal: PropTypes.bool,
+        show_post_advanced_settings_modal: PropTypes.string,
```
Comment (gl2748) [inline src/app/components/modules/Modals.jsx]: Are we sure this should be a string?

File: src/app/components/modules/Modals.jsx [hunk 2]
```diff
@@ -30,6 +31,7 @@ class Modals extends React.Component {
         show_transfer_modal: false,
         show_confirm_modal: false,
         show_login_modal: false,
+        show_post_advanced_settings_modal: '',
```
Comment (gl2748) [inline src/app/components/modules/Modals.jsx]: Why not bool?

Comment (gl2748) [reply src/app/components/modules/Modals.jsx]: Ah I see because it is the formId that is saved here.

Comment (eonwarped) [reply src/app/components/modules/Modals.jsx]: Ah yes, that's odd, I remember putting in a // formId comment there, but maybe it's somewhere else

**Comment by gl2748** [comment]
Closing this, while the OG PR is still in play.

**Comment by roadscape** [comment]
Looking good, need @pkattera to sign-off

**Comment by pkattera** [comment]
I made some minor copy + CSS changes and also added English strings to the translation files to avoid errors when viewing the new strings from other languages.

**Comment by roadscape** [comment]
Unexpected behavior:

1. In settings, both options set to 'Decline Payout'
2. Go to `/submit.html`, press Reset. "Rewards: Decline Payout" shown as expected.
3. Click 'Advanced Settings', shows `Default: 50% SBD / 50% SP` unexpectedly.

**Comment by eonwarped** [comment]
@roadscape  good catch. I sent a PR to address these.

**Comment by roadscape** [comment]
This is looking great, just waiting on feedback if we want this exposed on comments.

**Comment by roadscape** [comment]
@eonwarped could you please make the following changes:
 - for comments, do not expose advanced settings
 - when a non-default comment payout option is selected in settings, display the current setting and link to Settings page for adjustments. otherwise do not render anything.

We may need to expose these options later (perhaps through a power user option), but the vast majority of the value of the Advanced Settings dialog is on top-level posts. It's important to keep the comments interface as simple as possible for users who have not changed defaults, and the changes requested above should accomplish this.

**Comment by eonwarped** [comment]
Done, linked PR via reference.

**Comment by roadscape** [comment]
LGTM, merged. Thanks for this contribution @eonwarped!

**Comment by rebuildman** [comment]
I made an update on my capacitor, now I get the following error, commenting on any post, which makes it impossible to comment, Transaction broadcast error.
  "RPCError: Assert Exception: beneficiaries.size (): Must specify at least one beneficiary"

benefiarios
 
error
 
How can I fix it?

**Comment by eonwarped** [comment]
@rebuildman This was a much earlier version of the advanced setting module with beneficiary. You can find my newest changes here: https://github.com/steemit/condenser/pull/2700

**Comment by rebuildman** [comment]
@eonwarped  thank you !

**Pull request merged**